addHack.js

```diff
@@ -28,6 +28,10 @@ export const add = async (event, context) => {
 
   return {
     statusCode: 200,
+    headers: {
+      'Access-Control-Allow-Origin': '*',
```

Comment:
We should not be hardcoding `*` in our code. We should be setting this from environment variables.

I see it's this way in the other ones, so I propose we raise a ticket to fix this security issue.